src/core/embedders/CustomFontEmbedder.ts

```diff
@@ -82,6 +82,15 @@ class CustomFontEmbedder {
     return (1000 * height) / (yTop - yBottom);
   }
 
+  getFontMetrics() {
+    return {
+      ascent: this.font.ascent,
+      descent: this.font.descent,
+      lineGap: this.font.lineGap,
+      bbox: this.font.bbox,
+    };
+  }
+
```

Comment:
Please see my previous comment about setting fields in the constructor instead of adding a new method. I think the same thing applies to the font embedders themselves.